qiskit_dynamics/arraylias/register_functions/asarray.py

```diff
@@ -27,10 +27,6 @@ def register_asarray(alias):
     def _(arr):
         return np.asarray(arr)
 
-    @alias.register_fallback(path="asarray")
-    def _(arr):
-        return np.asarray(arr)
-
```

Comment:
I feel we can remove this fallback functions.

Reply:
Yes this seems okay since we have registered an `asarray` for every library.